tests/helpers.jsx

```diff
@@ -189,3 +189,23 @@ export class FakeBraintreeClientError {
     callback("I'm some error");
   }
 }
+
+export function assertNotificationType(action, notificationType) {
+  assert.equal(action.type, actionTypes.ADD_NOTIFICATION,
+               'action.type should be ADD_NOTIFICATION');
+  assert.equal(action.data.type, notificationType,
+               'action.data.type should be ' + notificationType);
+}
+
+export function assertNotificationTextContains(action, textSubString) {
+  assert.ok(typeof action.data.text === 'string',
+            'action.data.text should be a string');
+  assert.include(action.data.text, textSubString,
+                 'action.data.text should include substring');
+}
+
+export function assertNotificationErrorCode(action, errorCode) {
+  assertNotificationType(action, 'error');
+  assert.equal(action.data.errorCode, errorCode,
+               'action.data.errorCode must match errorCode');
```

Comment:
I don't think you need a third argument here. Without it, the message will log the actual and expected value which is more useful.